projects/worldskills-angular-lib/src/lib/i18n/en.json.ts

```diff
@@ -1,4 +1,8 @@
 export const en = {
```

Comment:
We will need to add these keys into `https://github.com/worldskills/worldskills-angular-lib/blob/main/projects/worldskills-angular-lib/assets/i18n/%20lazy-loaded/en.json` as well.
This is a slight downside to having isolated translations in libraries (for now).